N'affiche que les trois points du menu des message sur mobile

AmauryCarrade opened the pull request:
Tout est dans le titre.

### Contrôle qualité

1. `make run`
2. Vérifier sur un affichage mobile (sur un vrai téléphone ou sur la vue mobile du navigateur) que le bouton est réduit à trois points.
3. Vérifier que sur un ordi, le bouton est toujours complet.

AmauryCarrade committed:
N'affiche que les trois points du menu des message sur mobile

philippemilink commented:
Rapport de QA
----------------------
(sur la bêta)

**OK**, mais en effet, je rajouterais un peu de graisse sur les 3 points, et peut-être pas seulement sur la version mobile.

AmauryCarrade commented:
Pas que sur mobile, tu penses ? J'ai peur que ce soit trop “voleur d'attention” s'il y a le texte “Citer” à côté

AmauryCarrade committed:
Les points du menu glisse désormais

- La graisse de la police des points du menu a été augmentée, pour
  plus de lisibilité.

AmauryCarrade commented:
Fait sur les deux. En effet c'est mieux.

philippemilink commented:
Rapport de QA
---------------------

**OK** :heavy_check_mark: et c'est bien mieux comme ça !
